exonum-java-binding/service-archetype/src/main/resources/archetype-resources/start-testnet.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
```

Comment:
Basically, a simplified 'start-node.sh':

```patch
--- cryptocurrency-demo/start-node.sh   2020-02-28 11:49:52.000000000 +0200
+++ service-archetype/src/main/resources/archetype-resources/start-testnet.sh   2020-02-28 17:46:14.000000000 +0200
@@ -1,6 +1,14 @@
 #!/usr/bin/env bash
 
-# prints a section header
+# Starts a test Exonum network with a single validator node.
+#
+# The network files are placed in `$PWD/testnet` directory, which is cleared at the beginning.
+# The Java service artifacts must be placed into 'testnet/artifacts' directory.
+#
+# See https://exonum.com/doc/version/latest/get-started/java-binding/#node-configuration
+# for extra information.
+
+# Prints a section header
 function header() {
     local title=$1
     local rest="========================================================================"
@@ -9,56 +17,45 @@
     echo
 }
 
+# Fail immediately in case of errors and/or unset variables
 set -eu -o pipefail
 
-header "DETECTING ENVIRONMENT"
-
 # Check whether the Exonum Java App is installed.
-EXONUM_JAVA_APP="exonum-java"
-command -v ${EXONUM_JAVA_APP} >/dev/null 2>&1 || { echo >&2 "Please install the Exonum Java App and make sure that 'exonum-java' binary is available via PATH. Aborting."; exit 1; }
-
-# Find the latest version of artifact and build if not found.
-ARTIFACT_PATH="$(find ./target -type f -name 'exonum-java-binding-cryptocurrency-demo-*-artifact.jar')"
-
-if [ -z "${ARTIFACT_PATH}" ];
-then
-    mvn package
-fi
-ARTIFACT_PATH="target"
-echo "ARTIFACT_PATH=${ARTIFACT_PATH}"
+command -v exonum-java >/dev/null 2>&1 || { echo >&2 "Please install the Exonum Java App and make sure that 'exonum-java' binary is available via PATH. Aborting."; exit 1; }
 
 # Clear test dir.
-rm -rf testnet
-mkdir testnet
+TESTNET_PATH="$PWD/testnet"
+rm -rf "$TESTNET_PATH"
+mkdir "$TESTNET_PATH"
 
 # Enable predefined native logging configuration,
 # unless it is already set to any value (incl. null)
 export RUST_LOG="${RUST_LOG-error,exonum=info,exonum-java=info,java_bindings=info}"
 
-header "GENERATE COMMON CONFIG"
-${EXONUM_JAVA_APP} generate-template \
+header "1 Generate Template Config"
+exonum-java generate-template \
     --validators-count=1 \
     --supervisor-mode simple \
-    testnet/common.toml
+    "$TESTNET_PATH"/common.toml
 
-header "GENERATE CONFIG"
-${EXONUM_JAVA_APP} generate-config \
-    testnet/common.toml \
-    testnet \
+header "2 Generate Node Private and Public Configs"
+exonum-java generate-config \
+    "$TESTNET_PATH"/common.toml \
+    "$TESTNET_PATH" \
     --no-password \
     --peer-address 127.0.0.1:5400
 
-header "FINALIZE"
-${EXONUM_JAVA_APP} finalize \
-    testnet/sec.toml \
-    testnet/node.toml \
-    --public-configs testnet/pub.toml
-
-header "START TESTNET"
-${EXONUM_JAVA_APP} run \
-    --node-config testnet/node.toml \
-    --artifacts-path ${ARTIFACT_PATH} \
-    --db-path testnet/db \
+header "3 Finalize Configuration"
+exonum-java finalize \
+    "$TESTNET_PATH"/sec.toml \
+    "$TESTNET_PATH"/node.toml \
+    --public-configs "$TESTNET_PATH"/pub.toml
+
+header "4 Start Test Network"
+exonum-java run \
+    --node-config "$TESTNET_PATH"/node.toml \
+    --artifacts-path "$TESTNET_PATH"/artifacts \
+    --db-path "$TESTNET_PATH"/db \
     --master-key-pass pass \
     --public-api-address 127.0.0.1:3000 \
     --private-api-address 127.0.0.1:3010 \
```